rust/rust.cargo/nbproject/project.properties

```diff
@@ -15,5 +15,6 @@
 # specific language governing permissions and limitations
 # under the License.
 
+release.external/toml-java-1.1.1.jar=modules/ext/toml-java-1.1.1.jar
 javac.source=1.8
```

Comment:
setting this to 11 should hopefully make the test pass
```
javac.source=11
javac.target=11
```

failure:
```
testCheckClassFiles: org.netbeans.core.validation.ValidateClassFilesTest

junit.framework.AssertionFailedError: 
 found version 55.0 in net/vieiro/toml/TOMLSimpleQuery.class in D:\a\netbeans\netbeans\nbbuild\netbeans\rust\modules\ext\toml-java-1.1.1.jar
 found version 55.0 in net/vieiro/toml/TOMLParser.class in D:\a\netbeans\netbeans\nbbuild\netbeans\rust\modules\ext\toml-java-1.1.1.jar
 found version 55.0 in net/vieiro/toml/TOMLJSON.class in D:\a\netbeans\netbeans\nbbuild\netbeans\rust\modules\ext\toml-java-1.1.1.jar
...
```

Reply:
Thanks, @mbien . Yep, I'll try to send an update later during the day/weekend.